qa/L0_client_memory_growth/test.sh

```diff
@@ -56,8 +56,9 @@ source ../common/util.sh
 # Set the number of repetitions in nightly and weekly tests
 # Set the email subject for nightly and weekly tests
 if [ "$TRITON_PERF_WEEKLY" == 1 ]; then
-    REPETITION_CPP=2000000
-    REPETITION_PY=2400000
+    # Run the test for approximately 7 hours
```

Comment:
All tests cumulatively run for 7 hours or does each test case run for 7 hours? I think it would be better to mention the time for each test case that corr. to the reps for that test case.